CMakeLists.txt

```diff
@@ -109,27 +112,30 @@ function(trusted_firmware_build)
                -DTFM_TOOLCHAIN_PREFIX=${TFM_TOOLCHAIN_PREFIX}
     BUILD_ALWAYS True
     USES_TERMINAL_BUILD True
-    BUILD_BYPRODUCTS ${VENEERS_FILE} ${PSA_API_NS_PATH}
```

Comment:
Why are we running the install step also ?

Why not disable the install stage with
```
 INSTALL_COMMAND ""
```
because you are already using:
```
${ZEPHYR_TFM_MODULE_DIR}/trusted-firmware-m/interface/include
```
in line 129.

So you only need:
```
${TFM_BINARY_DIR}/generated/interface/include/psa_manifest
```

Reply:
The install step is needed for the signing commands. They access the bin files via there. That can probably be fixed later, but is IMO out of scope of this PR.

Reply:
I changed the path anyways.

Reply:
> The install step is needed for the signing commands. They access the bin files via there. 
Thanks for the explanation.
Could you make a comment on the ExternalProject, why the install step is needed :)